src/tcp_sock.rs

```diff
@@ -251,59 +238,24 @@ impl Inner {
     //                 will be attempted in the next write schedule.
     //   - Err(error): there was an error while writing to the socket.
```

Comment:
It was not the main focus on this PR. We have a separate task for doc improvements: https://github.com/maidsafe/socket-collection/issues/6 , so that's when I was planning to do it ;)